packages/contracts/contracts/package.json

```diff
@@ -0,0 +1,18 @@
+{
+    "name": "@semaphore-extensions/contracts",
+    "version": "0.1.0",
+    "description": "Semaphore extensions Contracts to manage anonymous voting and whistleblowing",
```

Comment:
Remember to update this description based on the comment in the README file above.

Reply:
Please remember to update this description.